pkg/userregistry/identitymapper/strategy_add.go

```diff
@@ -33,7 +34,12 @@ func (s *StrategyAdd) UserForNewIdentity(ctx context.Context, preferredUserName
 		desiredUser := &userapi.User{}
 		desiredUser.Name = preferredUserName
 		desiredUser.Identities = []string{identity.Name}
-		s.initializer.InitializeUser(identity, desiredUser)
+
+		err = s.initializer.InitializeUser(identity, desiredUser)
```

Comment:
This error is unused later so you can just paste it into the condition. Also, don't rewrite the err from the wrapping block